upgrade/UPGRADE-v13.0.0-dev.md

```diff
@@ -51,3 +51,10 @@ There you can find links to upgrade notes for other versions too.
     - already existing articles with this placement will be migrated to `Shopsys\FrameworkBundle\Model\Article\Article::PLACEMENT_NONE` placement
         - if you intend to keep using this placement, you can skip DB Migration `Shopsys\FrameworkBundle\Migrations\Version20230907132822` 
     - see #project-base-diff to update your project
+- allow quick searching in promo codes ([#2786](https://github.com/shopsys/shopsys/pull/2786))
+    - method `Shopsys\FrameworkBundle\Controller\Admin\PromoCodeController::listAction` changed its interface:
```

Comment:
Should we also mention the change of PromoCodeGridFactory::create?

Reply:
PromoCodeGridFactory::create in the framework package is unchanged and the project-base implementation is covered by #project-base-diff